README-automated-testing.md

```diff
@@ -2,40 +2,14 @@
 
 ## Installation test
 
-Github Workflow is used to test that this package can be installed. See `.github`.
+GitHub Action workflows test that this package can be installed. See the workflows in `.github/workflows`.
 
-## Rector automated functional tests using Behat
+## Rector functional testing
 
-Behat (the Php version of Cucumber) is used to run automated tests.
+The functional test takes the `rector_examples` directory in this package and copies it to a Drupal code base.
 
-This uses Linux / MacOS commands, so they need to be run from that environment.
+The workflow then runs `vendor/bin/rector process web/modules/custom/rector_examples` to apply all of the appropriate Rector rules.
 
-These tests assume that this repository is installed as a local composer package. This is necessary, because we need a full Drupal site to run the Rector tests.
+Then, the `diff` command is run to verify the changes match the expected results as found in the `rector_examples_updated` directory.
 
```

Comment:
@agentrickard Let me know if these changes work.

Try using the "Rich diff" view for better grokking.